turf/api/turf.api

```diff
@@ -29,18 +25,10 @@ public final class org/maplibre/spatialk/turf/grids/Grids {
 }
 
 public final class org/maplibre/spatialk/turf/measurement/Measurement {
-	public static final synthetic fun __area (Lorg/maplibre/spatialk/geojson/Geometry;)D
-	public static final synthetic fun __distance (Lorg/maplibre/spatialk/geojson/Position;Lorg/maplibre/spatialk/geojson/Position;)D
-	public static final synthetic fun __length (Lorg/maplibre/spatialk/geojson/Geometry;)D
-	public static final synthetic fun __pointToLineDistance (Lorg/maplibre/spatialk/geojson/Position;Lorg/maplibre/spatialk/geojson/LineString;)D
-	public static final synthetic fun __rhumbDistance (Lorg/maplibre/spatialk/geojson/Position;Lorg/maplibre/spatialk/geojson/Position;)D
```

Comment:
gave these synthetic functions `asArea` or `asLength` suffixes; user never sees it but at least that's more clear what it is in the abi